wonnx-cli/src/text.rs

```diff
@@ -0,0 +1,134 @@
+use std::borrow::Cow;
+use std::path::Path;
+use tokenizers::models::wordpiece::WordPieceBuilder;
+use tokenizers::normalizers::BertNormalizer;
+use tokenizers::pre_tokenizers::bert::BertPreTokenizer;
+use tokenizers::processors::bert::BertProcessing;
+use tokenizers::utils::padding::{PaddingDirection::Right, PaddingParams, PaddingStrategy::Fixed};
+use tokenizers::utils::truncation::TruncationParams;
+use tokenizers::utils::truncation::TruncationStrategy::LongestFirst;
+use tokenizers::{AddedToken, EncodeInput, Encoding, InputSequence, Tokenizer};
+use wonnx::utils::Shape;
+
+use crate::types::{NNXError, Tensor};
+pub struct BertTokenizer {
+    pub tokenizer: Tokenizer,
+}
+
+pub struct BertEncodedText {
+    pub encoding: Encoding,
+}
+
+impl BertTokenizer {
+    pub fn new(vocab_path: &Path) -> BertTokenizer {
+        let wp_builder = WordPieceBuilder::new()
```

Comment:
I feel that this is sort of part of a broader `preprocessing step` that we should probably migrate either to a new crate or a sub-part of the main crate. Not necessarily in this PR.